lib/rucio/core/authentication.py

```diff
@@ -214,7 +215,7 @@ def get_auth_token_gss(account, gsstoken, appid, ip=None, *, session: "Session")
 
 
 @transactional_session
-def get_auth_token_ssh(account, signature, appid, ip=None, *, session: "Session"):
+def get_auth_token_ssh(account: InternalAccount, signature: Union[str, bytes], appid: str, ip: Optional[str] = None, *, session: "Session") -> Optional[TokenDict]:
```

Comment:
I think `Union[str, bytes]` may warrant one additional look. As far as I can tell, the current calls of this functions use `bytes`. So why keep supporting `str`?